bam_masterdata/cli/run_parser.py

```diff
@@ -263,3 +265,41 @@ def run_parser(
             logger.info(
                 f"Linked child {child_openbis_id} to parent {parent_openbis_id} in collection {collection_name}."
             )
+
+
+def run_parser_withthreading(
+    openbis=None,
+    space_name: str = "",
+    project_name: str = "PROJECT",
+    collection_name: str = "",
+    files_parser: dict = {},
+    collection_type: str = "COLLECTION",
+    timeout: int = 300,
+    max_workers: int = 1,
+) -> None:
```

Comment:
maybe we could return the time it took to the multi thread executor of run_parser() instead of nothing?